- remove -march=native: it is managed by CPUTYPE in /etc/make.conf

--- Makefile.orig	2023-10-24 16:35:38 UTC
+++ Makefile
@@ -25,7 +25,7 @@ VERSION = 0.4.1
 CPPFLAGS += -MMD -MP -DVERSION=\"$(VERSION)\"
 CPPFLAGS += $(shell pkgconf --cflags freetype2)
 CXXFLAGS += -O2 -Wall
-CXXFLAGS += -march=native
+#CXXFLAGS += -march=native
 LDLIBS += -lclthreads -lclxclient -lpthread -ljack -lXft -lX11 -lrt
 
 
